gc3libs/__init__.py

```diff
@@ -976,7 +976,7 @@ def __init__(self, arguments, inputs, outputs, output_dir, **extra_args):
         self.join = extra_args.pop('join', False)
         self.stdin = extra_args.pop('stdin', None)
         if self.stdin and (self.stdin not in self.inputs):
-            self.inputs[self.stdin] = os.path.basename(self.stdin)
+            self.inputs[self.stdin] = os.path.abspath(self.stdin)
```

Comment:
Nope. This was correct. See the documentation for the `.inputs` attribute of `Application` classes in <https://gc3pie.readthedocs.io/en/master/programmers/api/gc3libs.html#gc3libs.Application>
-- the *key* of the inputs dictionary is the local path.